### PR TITLE
Redesign form

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #44

### CODE REVIEW

File: views/place-form.hbs [hunk 1]
```diff
@@ -7,39 +7,77 @@
       <h1>{{localText.placesAdd}}</h1>
       <input type="hidden" name="_method" value="post">
   {{/if}}
+  <span> Details below are required in at least one of Arabic and English.</span>
```
Comment (m4v15) [inline views/place-form.hbs]: This needs to be in the language file

**Review by m4v15 — CHANGES_REQUESTED** [review]
Also remove the height:100% from the css file

File: views/place-form.hbs [hunk 2]
```diff
@@ -7,39 +7,77 @@
...
-      <input type="text" name="name" required value="{{place.local.name}}">
-    {{else}}
-      <input type="text" name="name" required>
-    {{/if}}
-  </label>
+  <fieldset name="en" class="langFieldset">
+    <legend>{{localText.english}}</legend>
+    <label>{{localText.placeName}}
+      {{#if place.local.name}}
+      <input type="text" name="name_en" required value="{{place.local.name}}">
```
Comment (jwld) [inline views/place-form.hbs]: All these inputs need indenting inside the if statements :v:

Comment (Suuuuha) [reply views/place-form.hbs]: oh come on

Comment (jwld) [reply views/place-form.hbs]: lololololol get to work

**Review by jwld — CHANGES_REQUESTED** [review]
Indentation

**Comment by Suuuuha** [comment]
@JWLD @m4v15

**Review by jwld — APPROVED** [review]

**Review by m4v15 — APPROVED** [review]

**Pull request merged**